app/components/progress_tab/breakdown_table_component.html.erb

```diff
@@ -1,5 +1,5 @@
 <div>
-  <div class="d-flex ai-center jc-space-between <% if include_bottom_border %>pb-3px bb-grey-mid<% end %> mb-4px">
```

Comment:
Is `include_bottom_border` no longer required? Can we clean it up?

Why was this conditional here in the first place?

Reply:
We need this `include_bottom_border` condition. 

This was added to conditionally add a border to the `Transgender` row under the heading `Hypertension and diabetes`. By mistake, I added the condition for the `Hypertension and diabetes` row as well. 

<img width="250" alt="Screenshot 2022-11-15 at 6 56 48 PM" src="https://user-images.githubusercontent.com/32141642/201931250-667864b9-9fe8-4c59-b1f5-c1a25d466e3b.png">